kvmagent/kvmagent/plugins/storage_device.py

```
@@ -266,10 +291,61 @@ def start(self):
         http_server.register_async_uri(self.RAID_LOCATE_PATH, self.raid_locate)
         http_server.register_async_uri(self.RAID_SELF_TEST_PATH, self.drive_self_test)
         http_server.register_async_uri(self.GET_MULTIPATH_TOPOLOGY_PATH, self.get_multipath_topology)
+        http_server.register_async_uri(self.HBA_SCAN_PATH, self.hba_scan)
 
     def stop(self):
         pass
 
+    @kvmagent.replyerror
+    def hba_scan(self, req):
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
```

Ruff

Local variable `cmd` is assigned to but never used

Remove assignment to unused variable `cmd`

(F841)